cypress/integration/computation_old_combinations.spec.ts

```diff
@@ -0,0 +1,66 @@
+describe("ComputationMenu", () => {
+    beforeEach(() => {
+        cy.clearLocalStorage();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)